services/chat/api.go

```diff
@@ -353,7 +350,6 @@ func (chat *Chat) populateCommunityFields(community *communities.Community) erro
 	chat.Permissions = commChat.Permissions
 	chat.Emoji = commChat.Identity.Emoji
 	chat.Name = commChat.Identity.DisplayName
-	chat.Color = commChat.Identity.Color
```

Comment:
The color is already handled by the Chat color coming from the previous call. Setting it here again sometimes came up with an empty color.